lib/surrealist/builder.rb

```diff
@@ -8,10 +8,12 @@ class Builder
 
     # @param [Carrier] carrier instance of Surrealist::Carrier
     # @param [Hash] schema the schema defined in the object's class.
+    # @param [Hash] aliases the aliases map defined in the object's class.
     # @param [Object] instance the instance of the object which methods from the schema are called on.
-    def initialize(carrier:, schema:, instance:)
-      @carrier = carrier
-      @schema = schema
+    def initialize(carrier:, schema:, aliases: {}, instance:)
```

Comment:
I think `{}` part in `aliases: {}` is redundant, as you already do `instance.class.instance_variable_get(ALIASES_INSTANCE_VARIABLE) || {}` in `Surrealist.find_aliases()`